translate/toml.go

```diff
@@ -13,6 +15,18 @@ func (s Content) ToToml() string {
 	return result
 }
 
+// FromToml reads in a *.toml file and returns all mappings.
+func FromToml(s string) (c map[string]interface{}, err error) {
+	c = map[string]interface{}{}
+	//regex := regexp.MustCompile(`(?<=^[+]{3})((?:\s*.+=.+\s*)+)(?=^[+]{3})`)
+	regex := regexp.MustCompile(`((?:.+=.+\s*)+)`)
```

Comment:
The commented out regex is there to draw discussion: What I have on `ln22` does work for default-case, however, I want it to depend on the actual syntax of +++ or ---. What's on `ln21` works on the internet, but is not recognized by this regex library... Suggestions? Concerns?